lib/api/src/grpc/conversions.rs

```diff
@@ -528,7 +528,18 @@ impl From<NamedVectors> for HashMap<String, segment::data_types::vectors::Vector
         vectors
             .vectors
             .into_iter()
-            .map(|(name, vector)| (name, vector.data.into()))
+            .map(|(name, vector)| {
```

Comment:
Looks strange. The same convertion you may find from grpc vector into rest vector (`convertions.rs`, line 450). Try to understand why `into()` didn't work here

Reply:
@agourlay it seems change:
```
- .map(|(name, vector)| (name, vector.data.into()))
+ .map(|(name, vector)| (name, vector.into()))
```
Is enought